### PR TITLE
updating readme.md with instructions on .env, and adding .env_sample

### PR DESCRIPTION
**Pull request by Banjerr**
https://github.com/sendgrid/sendgrid-nodejs/issues/545

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=550) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=550) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=550) it.</sub>

File: README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ We appreciate your continued support, thank you!
 
 This library is broken up into several packages as a monorepo so that you only need to install the packages necessary for your use case. This README contains information pertaining to all packages. For examples on how to get started quickly, head over to the READMEs of each individual package (linked and described below), which includes detailed examples.
 
+## .env file
+
+You will find a file in the root of the project called .env_sample. You can run `mv .env_sample .env` or manually rename the file to `.env`; be sure to add your [API key](https://sendgrid.com/docs/User_Guide/Settings/api_keys.html#-Creating-an-API-key).
```
Comment (mbernier) [inline README.md]: It is better to copy the file and then edit it. This prevents subsequent updates to the dependency don't throw errors or get weird on someone's production server.

Can you please modify this from 
```
mv .env_sample .env
```
to 
```
cp .env_sample .env
```

Comment (Banjerr) [reply README.md]: @mbernier thanks for the feedback, i've updated the readme.md with your request

Comment (mbernier) [reply README.md]: Awesome, thank you!

**Comment by thinkingserious** [comment]
Hello @Banjerr,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mbernier** [comment]
@Banjerr 
If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/hacktoberfest-follow-up) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Pull request merged**